docs/docs/guides/deployment/hydra-oauth2/hydra-oauth2.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Use Ory Hydra as OAuth2 provider
+---
+
+A requirement to run your own Chainloop instance, is to have a way of authenticate users. One of the most common ways to do this is by using OAuth2.
+Ory Hydra is a popular OAuth2 provider that can be used to authenticate users in your Chainloop instance once connected to an external OIDC provider.
```

Comment:
> once connected to an external OIDC provider.

not sure I understand this part. Does this mean that hydra needs to be connected to an external OIDC? Or that Hydra is an OIDC IDp?

Reply:
I've removed the sentence but, Hydra needs to be connected to an external IDP